artshow/admin.py

```diff
@@ -591,3 +594,50 @@ def print_cheques(self, request, cheqs):
 
 
 admin.site.register(ChequePayment, ChequePaymentAdmin)
+
+
+@admin.register(SquarePayment)
+class SquarePaymentAdmin(admin.ModelAdmin):
+    @admin.display(description='Artist')
+    def clickable_artist(self, obj):
+        return format_html('<a href="{}">{}</a>',
+                           reverse('admin:artshow_artist_change',
```

Comment:
what is reverse doing here?

Reply:
Reverse takes a reference to a route and generates a (relative) URL which will resolve to the route. In this example it will generate a URL which links to the admin page for editing the given artist. In this case I think Django now supports this particular case as a built-in feature: https://docs.djangoproject.com/en/4.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_display_links

Reply:
Never mind, that's not what `list_display_links` does.